CryptoCurrency/start.sh

```diff
@@ -0,0 +1,2 @@
+docker build --tag crypto-deephaven/grpc-api .
```

Comment:
I think we should try to scope the namespace to the GH org this example is in. Right now, it's deephaven. Eventually, it may be https://github.com/deephaven-examples.

`deephaven/coingecko-grpc-api`, or `deephaven-examples/coingecko-grpc-api` would be my suggestion.

Reply:
@devinrsmith better?